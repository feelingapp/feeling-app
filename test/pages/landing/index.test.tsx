import { shallow } from "enzyme"
import Landing from "../../../src/pages/landing"

describe("Landing: ", () => {
  it("matches snapshot", () => {
    const wrapper = shallow(<Landing/>)
    expect(wrapper).toMatchSnapshot()
  })
})
