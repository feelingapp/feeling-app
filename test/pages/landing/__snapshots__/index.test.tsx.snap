// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing:  matches Header snapshot 1`] = `
<HeaderContainer>
  <test-file-stub />
  <Links>
    <Link
      href="https://twitter.com/feelingapp"
      rel="noreferrer noopener"
      target="_blank"
    >
      <test-file-stub />
      <LinkText>
        Twitter
      </LinkText>
    </Link>
    <Link
      href="mailto:hello@feeling.to"
    >
      <test-file-stub />
      <LinkText>
        Contact
      </LinkText>
    </Link>
  </Links>
</HeaderContainer>
`;

exports[`Landing:  matches Landing snapshot 1`] = `
<Fragment>
  <Head>
    <title>
      Feeling
    </title>
  </Head>
  <Container>
    <Header />
    <Main />
  </Container>
</Fragment>
`;

exports[`Landing:  matches Main snapshot 1`] = `
<MainContainer>
  <Title>
    Your feelings are about to get a whole lot smarter ðŸ˜Š
  </Title>
  <Input>
    <EmailInput
      placeholder="Your email"
      type="text"
    />
    <EarlyAccessButton>
      Get Early Access
    </EarlyAccessButton>
  </Input>
  <Description>
    Feeling is an intelligent journal and mood tracker that uses science to help you learn more about yourself.
    <List>
      <li>
        Journaling has shown to blah blah blah
      </li>
      <li>
        Write or talk about your day
      </li>
      <li>
        Get analysed and see how the people around you are feeling
      </li>
    </List>
  </Description>
  <picture>
    <PhoneImage
      alt="iPhone"
      src="/iphone_crop.png"
    />
  </picture>
</MainContainer>
`;
